packages/celer-web-app/src/core/map/MapEngine.ts

```diff
@@ -38,6 +42,10 @@ export class MapEngine {
 	}
 
 	private computeLine(line: DocLine, outIcons: MapIcon[], outLines: MapLine[]): void {
+		if(line.lineType === "DocLineSection"){
+			this.endCurrentPathIfNeed(outLines);
+			this.currentSection = line.sectionNumber;
+		}
 		if(line.lineType === "DocLineBanner" || line.lineType === "DocLineSection") {
```

Comment:
The second half of this `if` is already covered above

Reply:
Alright, I moved it around. I'm not entirely sure if that's what you meant though.

Reply:
I meant something like this
```
if (line.lineType === "DocLineBanner") {
return;
}

if (line.lineType === "DocLineSection") {
this.endCurrentPathIfNeed(outLines);
this.currentSection = line.sectionNumber;
return;
}
```
Then you don't have repeated conditions in the ifs